test/mix/tasks/phoenix.gen.model_test.exs

```diff
@@ -103,6 +112,18 @@ defmodule Mix.Tasks.Phoenix.Gen.ModelTest do
         assert file =~ "field :title, :string"
         assert file =~ "belongs_to :user, Phoenix.User"
       end
+
+      assert_raise Mix.Error, fn ->
```

Comment:
Please define a separate test where you simply use a dummy module defined in this file. :)